Basic_viewer/doc/Basic_viewer/CGAL/Graphics_scene.h

```diff
@@ -0,0 +1,87 @@
+
+namespace CGAL {
+
+/*!
+ \ingroup PkgBasicViewerClasses
+
+The class `Graphics_scene` stores points, segments, triangles, rays and lines. Elements can be added, possibly with associated colors. Non triangular faces can be directly added and are triangulated internally.
+
+\tparam BufferType the number type used for point coordinates: `float` by default.
+
+*/
+template <typename BufferType=float>
+class Graphics_scene {
+public:
+  /// adds the given point in the scene.
+  template <typename KPoint>
+  void add_point(const KPoint &p);
+
+  /// adds the given colored point in the scene.
+  template <typename KPoint>
+  void add_point(const KPoint &p, const CGAL::IO::Color &acolor);
```

Comment:
Unless the `a` prefix is used consistently I would drop it.

Reply:
done